src/libraries/Microsoft.PowerFx.Core/Public/IPowerFxScope.cs

```diff
@@ -29,5 +31,15 @@ public interface IPowerFxScope
         /// <param name="expression">The expression to convert for display.</param>
         /// <returns>The expression with display formatting applied.</returns>
         string ConvertToDisplay(string expression);
+
+        DefinitionsCheckResult CheckUserDefinedFunctions(string expression);
```

Comment:
>DefinitionsCheckResult [](http://example.com/codeflow?start=8&length=22)

this is API breaking. Can we avoid adding on this interface and just go through EditorContextScope.